src/main/kotlin/no/nav/familie/ks/sak/kjerne/eøs/felles/BehandlingId.kt

```diff
@@ -0,0 +1,5 @@
+package no.nav.familie.ks.sak.kjerne.eøs.felles
+
+data class BehandlingId(
```

Comment:
Ser at det er sånn i BA-også, men lurer på om denne bør kanskje flyttes til en annen pakke som er litt mer felles?
Hvis jeg husker rett så ble BehandlingId klassen laget for å wrappe rundt `long` for å unngå forvirrelser og blanding med personIdent og andre id'r. Så tenker at den kanskje bør flyttes ut av EØS, og til et mer felles sted 🤔

Reply:
Den er ikke i bruk enda i dag i ks. Jeg tenkte å skrive om til å bruke BehandlingId når jeg har kopiert over Valutakurser. Slik at det blir likt. Kan flytte den da